blog/12-07-2022-solidity-from-zero-to-hero/solidity-from-zero-to-hero.md

```diff
@@ -0,0 +1,468 @@
+# Solidity from Zero to Hero
+
+# Introduction
+
+Remix is one of the online programs that can be used to write and deploy the Solidity Smart contract on the Ethereum Blockchain. It is a powerful tool that can also be used to debug and prevent errors in the Solidity contracts. In this article, I’ll show you how to use Remix for writing and deploying, debugging Solidity smart contracts, and how to enter some Solidity commands. This tutorial assumes you are familiar with the basics of blockchain, the Ethereum platform, and programming languages like JavaScript, Python, and C++. If you're ready to start coding, then let's get started!
+
+# Background
+
+Solidity is a contract-oriented programming high-level language for writing and deploying Smart Contracts on the Ethereum blockchain.  It was designed to make it as user-friendly as possible by Gavin Wood—the co-founder of Ethereum.
+
+# The Course Outline
+
+Below is a list of what we’ll cover
+
+💡 Types of Solidity Contracts
+
+💡 Step by step of writing and running a Solidity Smart Contract on Remix
+
+💡 Step by Step on Debugging Smart contract using Remix Debugger
+
+💡 A glossary of some Solidity commands and how to run them
+
+# Introduction to Remix IDE
+
+To write and deploy a Solidity smart contract on Remix is a simple and interesting task if only the appropriate procedures are taken. Let’s take a practical step by writing and running a simple, smart contract as follows;
+
+1. Go to https://remix.ethereum.org/.
+2. A browser window will open displaying the Remix web-based development environment as below.
+    
+    ![remix IDE](images/1.png)
+    
+
+3. In the left column of the Remix window, expand the Solidity folder and then click on the Contracts folder.
+
+![Remix IDE](images/2.png)
+
+4 In the right column of the Remix window, click on the New file button.
+
+![Remix IDE](images/3.png)
+
+5. In the Name text box, enter HelloWorld. The name should end with .sol and not .tx
+    
+    ![Remix IDE](images/4.png)
+    
+
+# HelloWorld Program in Solidity
+
+The code below is a simple hello world code in Solidity Code.
+
+```solidity
```

Comment:
Please format the code properly here.
- Remove extra lines in between the code.
- Instead of tabs for code indentation use four spaces for consistency.